lib/msf/core/rpc/v10/rpc_module.rb

```diff
@@ -467,6 +467,36 @@ def rpc_options(mtype, mname)
     res
   end
 
+  # Assigns the appropriate dispatcher based on the `mod_type` passed
+  # @param [Msf::Module] mod_type
+  def dispatcher(mod_type)
+    case mod_type
+    when 'encoder'
+      Msf::Ui::Console::CommandDispatcher::Encoder
+    when 'exploit'
+      Msf::Ui::Console::CommandDispatcher::Exploit
+    when 'nop'
+      Msf::Ui::Console::CommandDispatcher::Nop
+    when 'payload'
+      Msf::Ui::Console::CommandDispatcher::Payload
+    when 'auxiliary'
+      Msf::Ui::Console::CommandDispatcher::Auxiliary
+    when 'post'
+      Msf::Ui::Console::CommandDispatcher::Post
+    when 'evasion'
+      Msf::Ui::Console::CommandDispatcher::Evasion
+    else
+      print_error("Unsupported module type: #{mod_type}")
+      return false
+    end
+  end
+
+  # Takes a module and selects a reasonable default payload and minimally configure it
+  # @param [Msf::Module] mod
+  def self.choose_payload(mod)
+    Msf::Ui::Console::CommandDispatcher::Exploit.choose_payload(mod)
+  end
```

Comment:
Is this dead code?

Reply:
Ah, yes it is. I had removed this, not sure how it snuck back in 😄